napalm_yang/translators/text.py

```diff
@@ -76,7 +82,7 @@ def _default_element_container(self, mapping, translation, replacing):
 
     def _xml_to_text(self, xml, text=""):
         for element in xml:
-            if element.tag == "command":
+            if element.tag == "command" and element.text is not None:
```

Comment:
For a container without `key_value` you'll get an empty `<command/>` element in the XML document. In that case `element.text` is None and the next line crashes.

I'll see if I can avoid creating the element in such cases.